libs/metagen/src/mdk_python/static/main.py.jinja

```diff
@@ -0,0 +1,14 @@
+{% if t_input != t_output %}
+from {{ type_mod_name }}_types import Struct, {{ t_input }}, {{ t_output }}
+{% else %}
+from {{ type_mod_name }}_types import Struct, {{ t_input }}
+{% endif %}
+
+def handle(inp: {{ t_input }}) -> {{ t_output }}:
+    # Write your logic here
+    raise Exception("not implemented")
+
+# Do not edit this function
+def {{ fn_name }}(raw_inp):
```

Comment:
can this be moved to the other file to have the following "convention"?
- x_types = do not edit/regenerated often
- x = do what ever you want, and you import what you need from types